scripts/setup-macos.sh

```diff
@@ -37,7 +37,7 @@ NPROC=$(getconf _NPROCESSORS_ONLN)
 COMPILER_FLAGS=$(get_cxx_flags $CPU_TARGET)
 
 DEPENDENCY_DIR=${DEPENDENCY_DIR:-$(pwd)}
-MACOS_DEPS="ninja cmake ccache protobuf icu4c boost gflags glog libevent lz4 lzo snappy xz zstd openssl@1.1"
```

Comment:
Lets not add additional dependencies for base macos install. We are in the process of moving most of these out of brew as  libraries installed this way cannot be cross compiled.